cocotb/config.py

```diff
@@ -156,6 +156,13 @@ def lib_name_path(interface, simulator):
     return library_name_path
 
 
+def findlibpython():
+    libpython_path = find_libpython.find_libpython()
+    if libpython_path is None:
+        sys.exit(1)
```

Comment:
Would exception be better?

Reply:
Printing nothing and having a non-0 exit code is exactly what `find_libpython` does. Because this is just a wrapper, I don't see why we shouldn't do the same thing.

There is a return code check and error message in the makefiles. I guess it doesn't matter where the error message comes from, but I still need to catch non-0 exit and then `$(error) in the makefiles.

Reply:
Was thinking about something like this 
```suggestion
        raise Exception('find_libpython was not able to find a libpython in the current Python environment')
```
but I see now it does not matter.